src/main/java/org/team1540/robot2018/Robot.java

```diff
@@ -172,6 +345,15 @@ public void testInit() {
 
   @Override
   public void robotPeriodic() {
+    SmartDashboard.putNumber("Elevator postion", elevator.getPosition());
```

Comment:
Should decide on a consistent naming scheme for these, as given the lack of organization in SmartDashboard/Shuffleboard, it can be terribly hard to find things.